pkg/config/config_factory_test.go

```diff
@@ -15,10 +15,11 @@
 package config
 
 import (
-	"go.opentelemetry.io/collector/config/configparser"
 	"os"
 	"testing"
 
+	"go.opentelemetry.io/collector/config/configloader"
```

Comment:
I think you remove the empty line at 22 and run `make fmt`,  and it show up alongside `go.opentelemetry.io/collector/service`

Reply:
Thx.